test/doro_web/api/entities_controller_test.exs

```diff
@@ -6,108 +6,81 @@ defmodule DoroWeb.Api.UserControllerTest do
-        |> json_response(200)
-
-      assert response == %{"message" => "created entity: tomcat"}
-
-      assert match?(
-               %Entity{
-                 behaviors: %{Doro.Behaviors.Visible => _},
-                 id: "tomcat",
-                 name: "tomcat",
-                 props: %{},
-                 proto: nil
-               },
-               Doro.World.get_entity("tomcat")
-             )
-    end
-
-    test "&create/2 creates a new entity with specified behaviors", %{
-      conn: conn,
-      params: params
+    test "creates a new entity with the right props", %{
+      conn: conn
     } do
-      extra_params = %{
-        behaviors: ["this", "turntable"]
+      params = %{
+        "behaviors" => [
+          %{ "type" => "exit", "destination_id" => "fire_escape"},
+          %{ "type" => "visible", "description" => "the description"}
+        ],
+        "id" => "new-thing",
+        "name" => "new thing",
+        "props" => %{"location" => "closet"}
       }
 
       conn
-      |> put("/api/entities", params |> Map.merge(extra_params))
+      |> put("/api/entities", params)
       |> json_response(200)
 
-      assert match?(
-               %Entity{
-                 behaviors: %{Doro.Behaviors.Visible => _, Doro.Behaviors.Turntable => _},
-                 id: "tomcat",
-                 name: "tomcat",
-                 props: %{},
-                 proto: nil
-               },
-               Doro.World.get_entity("tomcat")
-             )
-    end
-
-    test "&create/2 creates a new entity with the right props", %{
-      conn: conn,
-      params: params
-    } do
-      extra_params = %{
-        name: "the tomcat",
-        behaviors: nil,
-        props: %{location: "room", description: "this is the description"}
-      }
-
-      conn
-      |> put("/api/entities", params |> Map.merge(extra_params))
-      |> json_response(200)
-
-      entity = Doro.World.get_entity("tomcat")
-      assert entity.id == "tomcat"
-      assert entity.name == "the tomcat"
-      assert entity.name_tokens == MapSet.new(["the", "tomcat", "the tomcat"])
-      assert entity[:location] == "room"
+      entity = Doro.World.get_entity("new-thing")
+      assert entity.id == "new-thing"
+      assert entity.name == "new thing"
+      assert entity.name_tokens == MapSet.new(["new", "thing", "new thing"])
+      assert entity[:location] == "closet"
       assert Entity.has_behavior?(entity, Doro.Behaviors.Visible)
+      visible_behavior = entity |> Map.get(:behaviors) |> Map.get(Doro.Behaviors.Visible)
+      assert Map.get(visible_behavior, :description) == "the description"
+      exit_behavior = entity |> Map.get(:behaviors) |> Map.get(Doro.Behaviors.Exit)
+      assert Map.get(exit_behavior, :destination_id) == "fire_escape"
     end
   end
 
   describe "&update/2" do
     setup do
       Doro.World.load("priv_file://fixtures/turntable.json")
-      %{params: %{id: "turntable", behaviors: ["visible", "slot_machine"]}}
```

Comment:
Moving forward I'm not really sure if we care about the update method...  it's not used right now because once we have id, we overwrite the record.  But... i guess if we had more of a `patch` interface from the front (instead of sending the whole entity with each create/update, this could be useful.

for now... just leave it.  the tests are passing and the function works even if we never use it.